src/Stratis.Bitcoin.Features.SmartContracts.Wallet/ContractAccountHistoryModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
```

Comment:
Looks this this file and some others are duplicated between the two wallet projects. Was this intentional?

Reply:
They're nearly identical but with a different enum I believe is the only change. This little change forced all the concrete classes to change